images/pack-builder/install-pack.sh

```diff
@@ -6,18 +6,17 @@
 
 declare -r PACK_VERSION='0.1.0'
 
-if [[ "$OSTYPE" == "linux-gnu" ]]; then
-	packPlatform='linux';
-elif [[ "$OSTYPE" == "darwin"* ]]; then
-	packPlatform='macos';
-elif [[ "$OSTYPE" == "cygwin" || "$OSTYPE" == "msys" ]]; then
-	echo 'ERROR: `pack create-builder` is not implemented on Windows.'
-	exit 1
-else
-	echo 'ERROR: Could not detect compatible pack binary platform.'
-	exit 1
+packPlatform="$1"
```

Comment:
why not rely on the Bash variable OSTYPE?

Reply:
We did not want to duplicate these checks?